INSTALL.md

```diff
@@ -31,6 +31,62 @@ Alternatively, the debug target also work:
 
 `make debug`
 
+### Example build on a Debian 10 System
+
+Download a debian install image e.g. from https://www.debian.org/CD/netinst/
+and install debian 10 on a virtual or real machine. Create your user account
+during installation.
+
+Add your user to the sudoer group.
+
+```
+su -
+usermod -aG sudo <user>
+# now you need to logout and login again!
+```
+
+Install git.
+
+`sudo apt-get install git`
+
+Set your proxy variables (if necessarry) and clone the caprice repository.
+
+```
+export HTTP_PROXY="http://proxy.company.com:9999"
+export HTTPS_PROXY=$HTTP_PROXY
+
+git clone https://github.com/ColinPitrat/caprice32.git
+cd caprice32
+```
+
+Install g++.
```

Comment:
You could install everything in one command:

`sudo apt-get install g++ make libsdl1.2-dev libfreetype6-dev`